charts/kusion/templates/post-run-job.yaml

```diff
@@ -30,57 +31,20 @@ spec:
           command:
             - /bin/sh
             - -c
-            - >
```

Comment:
Reducing backoffLimit from 10 to 5 might increase the chance of job failure. Consider keeping it at 10 for better reliability.